<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>351</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>280</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Offset Safety</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> X1L =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> X1L =</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>351</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QPushButton" name="btn_OffsetSafety">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>280</y>
          <width>101</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Offset Safety</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x1l">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string> X1L =</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x2l">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string> X2L =</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string> X3L =</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x3l">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x3r">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>100</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string> X2R =</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string> X1R =</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x1r">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string> X3R =</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x2r">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>70</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>100</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Y3 =</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_y2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>70</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_y3">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>100</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Y2 =</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>40</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Y1 =</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_y1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_y3_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>230</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x2r_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>200</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_y1_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>170</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x1l_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x3r_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>230</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_y2_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>200</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x2l_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>200</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x3l_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>230</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_x1r_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>170</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Before offset</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>After offset</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
